src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -11173,19 +11167,12 @@ void Dummy()
             var compilation = CreateCompilation(new[] { source }, options: WithNonNullTypesTrue());
 
             compilation.VerifyDiagnostics(
-                 // (19,49): warning CS8608: Nullability of reference types in type doesn't match overridden member.
-                 //     public override event System.Action<string> E2;
-                 Diagnostic(ErrorCode.WRN_NullabilityMismatchInTypeOnOverride, "E2").WithLocation(19, 49),
-                 // (18,50): warning CS8608: Nullability of reference types in type doesn't match overridden member.
-                 //     public override event System.Action<string?> E1;
-                 Diagnostic(ErrorCode.WRN_NullabilityMismatchInTypeOnOverride, "E1").WithLocation(18, 50),
-                 // (26,49): warning CS8608: Nullability of reference types in type doesn't match overridden member.
```

Comment:
>// (26,49): warning CS8608: Nullability of reference types in type doesn't match overridden member. [](start = 17, length = 99)

The lack of warnings in B2 is unexpected. This is likely related to the improperly used ```IsImplicitlyDeclared``` check.

Reply:
Ah, I misread this test and assumed B2 was inheriting from B1. Yeah, that makes sense.